docs/install.md

```diff
@@ -0,0 +1,189 @@
+---
+sidebar_position: 1
+description: How to install Yazi on various operating systems.
+---
+
+# Installation
```

Comment:
We need to keep the h1 title name as same as its filename, should we change this to `Install` or change back the filename to `Installation`?